src/server.mjs

```diff
@@ -113,6 +114,36 @@ var limits = {
 };
 var upload = multer({ storage, limits });
 
+// runs every hour
+// deletes any files in the download directory past the set fileExpirationTime set in config
+cron.schedule('0 * * * *', () => {
+  console.log("cron scheduled check");
+  const currentTime = new Date().getTime();
+  // loop through these specified directories
+  for (const dir of [DOWNLOAD_DATA_PATH, UPLOAD_DATA_PATH]) {
+    fs.readdir(dir, (err, files) => {
+      
+      if (!files || files.length === 0) {
+        return;
+      }
+
+      files.forEach((file) => {
+        const filePath = path.join(dir, file)
+        // get file statistics
+        fs.stat(filePath, (statErr, stats) => {
+          const creationTime = stats.birthtime.getTime();
+          if (currentTime - creationTime >= config.fileExpirationTime) {
```

Comment:
When we properly support subdirectories, a parent directory might be expired before all its contents are, because the parent directory was created before the files in it were created. But it's not possible to delete the parent directory while there are things inside it. So we'd need to handle that somehow.

We might want to handle expiration at the chunk directory/individual file upload level, and remove whole chunk directories as a unit.